db/hello:1:greeter_spec.rb

```diff
@@ -0,0 +1 @@
+hello/1/greeter_spec.rb
```

Comment:
indeed i think was working on something else! will delete, i sort of saw it yesterday and thought hmmm 


----- Original Message -----

From: "Levi Cole" <notifications@github.com> 
To: "TeriPastorino/Blocitoff" <Blocitoff@noreply.github.com> 
Cc: "TeriPastorino" <tpasto@comcast.net> 
Sent: Wednesday, March 18, 2015 10:40:28 AM 
Subject: Re: [Blocitoff] wip redirect after login show user info (#3) 



In db/hello:1:greeter_spec.rb : 
> @@ -0,0 +1 @@
> +hello/1/greeter_spec.rb 


I'm not sure what's going on here, but I have a feeling making this file was an accident? 

— 
Reply to this email directly or view it on GitHub .